easybuild/easyconfigs/g/graph-tool/graph-tool-2.55-foss-2022a.eb

```diff
@@ -0,0 +1,57 @@
+# Author J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
```

Comment:
This might have a better fit in configuremakepythonpackage

Reply:
`ConfigureMakePythonPackage` uses a python command at the configure stage (see https://github.com/easybuilders/easybuild-easyblocks/blob/develop/easybuild/easyblocks/generic/configuremakepythonpackage.py#L60) and that fails with `unknown option --enable-openmp`.

Reply:
Doh, forgot about that...